elfpy/data/contract_interface.py

```diff
@@ -31,6 +31,12 @@ class HyperdriveAddressesJson:
     base_token: str = attr.ib()
 
 
+def camel_to_snake(camel_string: str) -> str:
```

Comment:
this belongs in a base utils folder/file.  i think we have a formatting.py it could live in.